src/server/scripts/EasternKingdoms/AlteracValley/boss_balinda.cpp

```diff
@@ -81,11 +81,11 @@ class boss_balinda : public CreatureScript
         {
             Talk(SAY_AGGRO);
             events.ScheduleEvent(EVENT_ARCANE_EXPLOSION, urand(5 * IN_MILLISECONDS, 15 * IN_MILLISECONDS));
-            events.ScheduleEvent(EVENT_CONE_OF_COLD, 8 * IN_MILLISECONDS);
-            events.ScheduleEvent(EVENT_FIREBOLT, 1 * IN_MILLISECONDS);
-            events.ScheduleEvent(EVENT_FROSTBOLT, 4 * IN_MILLISECONDS);
-            events.ScheduleEvent(EVENT_SUMMON_WATER_ELEMENTAL, 3 * IN_MILLISECONDS);
-            events.ScheduleEvent(EVENT_CHECK_RESET, 5 * IN_MILLISECONDS);
+            events.ScheduleEvent(EVENT_CONE_OF_COLD, 8s);
+            events.ScheduleEvent(EVENT_FIREBOLT, 1s);
+            events.ScheduleEvent(EVENT_FROSTBOLT, 4s);
+            events.ScheduleEvent(EVENT_SUMMON_WATER_ELEMENTAL, 3s);
+            events.ScheduleEvent(EVENT_CHECK_RESET, 5s);
```

Comment:
5 * IN_MILLISECONDS rly?

Reply:
Excuse me?

Reply:
@Faq : you need to be more specific if you think something is wrong.

Reply:
Nevermind